action.yml

```diff
@@ -44,6 +44,10 @@ runs:
     - shell: pwsh
       run: ${{ github.action_path }}/scripts/convertSecretsEnv.ps1
 
+    - name: Convert azure token into base64 encoded
+      run: |
+        echo "AZ_DEVOPS_BASE64_PAT=$(echo -n ${{ env.AZ_DEVOPS_PAT }} | base64)" >> $GITHUB_ENV
```

Comment:
You need to add a mask as well since this is sensitive.

Reply:
are we talking about "add-mask::"?

Reply:
added the mask.